features/welcome/components/WelcomePage.native.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
```

Comment:
As eslint puts it:

error  'React' must be in scope when using JSX  react/react-in-jsx-scope